deps/wazuh_testing/wazuh_testing/modules/fim/event_monitor.py

```diff
@@ -357,14 +357,25 @@ def callback_detect_file_deleted_event(line):
     return None
 
 
-def callback_restricted(line):
-    """ Callback that detects if a line in a log  if a file is ignored due to configured restrict tag.
+def callback_audit_cannot_start(line):
+    """ Callback that detects if a line shows whodata engine could not start and monitoring switched to realtime.
 
     Args:
         line (String): string line to be checked by callback in FileMonitor.
 
     Returns:
-        returns the entry that is being ignored.
+        Returns True if line matches.
+    """
+    match = re.match(r'.*Who-data engine could not start. Switching who-data to real-time.', line)
+    if match:
+        return True
+
+
+def callback_restricted(line):

```

Comment:
Improves method name is too generic.

Reply:
This callback was not added in my changes, it was moved because of and is being detected as a change. Renaming it would require to change the tests that use this:
- test_restrict_valid 
-  test_registry_restrict. 

Both tests are unrelated to this issue.